product_docs/docs/odbc_connector/13/05_edb-odbc_connection_properties.mdx

```diff
@@ -118,6 +118,7 @@ Select **Page 2** (in the upper-left corner of the Advanced Options dialog box)
 -   Select **Read Only** to prevent the driver from executing the following commands: `INSERT`, `UPDATE`, `DELETE`, `CREATE`, `ALTER`, `DROP`, `GRANT`, `REVOKE` or `LOCK`. Invoking the **Read Only** option also prevents any calls that use ODBC’s procedure call escape syntax (`call=procedure-name?`). By default, this option is off.
 -   Select **Show System Tables** to include system tables in the result set of the `SQLTables()` function. If the option is enabled, the driver includes any table whose name starts with `pg\_` or any of the prefixes listed in the **SysTablePrefixes** field of Page 1 of the Advanced Options dialog box. By default, this option is off.
 -   Select **Show sys/dbo Tables [Access]** to access objects in the `sys` schema and `dbo` schema through the ODBC data source. By default, this option is on.
+-   Select **Show sys/SQL Tables** to enable accessing database tables in a linked server with SQL server.
```

Comment:
I don't understand this as it is written. Is the option to enable accessing tables in linked servers with SQL commands or is the linked server running SQL Server? If the former, I don't think you need server after SQL. If the latter, I think server should be capitalized (SQL Server) although the bug descriptions say MS SQL. You should either use SQL Server, MS SQL, or MS SQL Server in both places. I'd go with SQL Server.

Reply:
changed to SQL Server and committed the changes